bentoml/server/bento_api_server.py

```diff
@@ -256,8 +261,19 @@ def wrapper():
             prediction_logger.info(request_log)
 
             response.headers["request_id"] = request_id
+
             return response
 
+        start_time = default_timer()
+
+        response = handle()
+
+        request_metric_time.labels(
```

Comment:
Is this something we can do at flask level for all endpoints?  with something like before_request, after_request

Reply:
Note that we don't need a `Summary` object for each API endpoint, they are already part of the label  - correct me if I'm wrong about this

Reply:
Yes, if we change the metric name to include the api name, then there is no need to add this `endpoint` label here.

Reply:
For flask level, I think that's possible. Do you have some concerns for doing it here, or are there some benefits to instrument this at flask level?